services/shared/src/types/zod.ts

```diff
@@ -5,9 +5,30 @@ import {
     MAX_LENGTH_OPTION,
     MIN_LENGTH_USERNAME,
     MAX_LENGTH_USERNAME,
+    MAX_LENGTH_BODY,
 } from "../shared.js";
 import { isValidPhoneNumber } from "libphonenumber-js";
 
+export const zodReportReason = z.union([
```

Comment:
No, use `z.enum(["offtopic", "spam"])`

don't use dash in the enum values btw.

Also change the values to the one I mentioned.